client/src/components/navbar/index.js

```diff
@@ -41,10 +39,7 @@ const Navbar = props => {
           />
         </ToolbarGroup>
         <ToolbarGroup>
-          <div className="navbarSearch">
-            <TextField hintText="Search" />
-          </div>
-          <RaisedButton label="Search" />
+          <NavbarForm onSubmit={fetchResults} />
```

Comment:
The naming of this component is confusing because in navbarForm.js We're exporting SearchBar.